web/webkit/src/test/scala/net/liftweb/http/SecurityRulesSpec.scala

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright 2011 WorldWide Conferencing, LLC
```

Comment:
I'll try and do that. This is why I strongly dislike these headers... so many of them are out of date anyway. < grump grump grump >